ProcessLib/HeatTransportBHE/BHE/BuildingPowerCurves.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
```

Comment:
```suggestion
```
It would be nice, if you check other includes too.

Reply:
Sorry about that, deleted.

Reply:
> It would be nice, if you check other includes too.
I mean all other includes in all other files too, not just here.

Reply:
Got it.